### ISSUE TITLE
`python3Packages.tifffile` build failure on x86_64-linux as of `cfcfea5d`

### ISSUE THREAD

**Issue by samuela** [NixOS/nixpkgs#161810]
## Issue description
Build of `python3Packages.tensorflow-datasets` failed on x86_64-linux as of cfcfea5dbe319af077fb01c872c52b121077ea82.

[build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/1897532785)

Maintainer cc: @ndl
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit cfcfea5dbe319af077fb01c872c52b121077ea82
2. Run `nix-build -A python3Packages.tensorflow-datasets`

## Technical details
```
- system: `"x86_64-linux"`
 - host os: `Linux 5.11.0-1028-azure, Ubuntu, 20.04.3 LTS (Focal Fossa)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.6.1`
 - nixpkgs: `/nix/store/6v2wg6ycxzrqwn5g2pxfvx5700vddq9a-source`
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/1897532785) that created this issue.
- Internal tag: 9007990010406861983

**Comment by samuela** [comment]
This looks like another tifffile failure:
```
2022-02-25T10:08:07.7267313Z error: builder for '/nix/store/hbbljss5hmxkx80ni3gybdk4adkvp544-python3.9-tifffile-2022.2.9.drv' failed with exit code 137;
2022-02-25T10:08:07.7270827Z        last 10 log lines:
2022-02-25T10:08:07.7274175Z        > ........................................................................ [ 75%]
2022-02-25T10:08:07.7274537Z        > ........................................................................ [ 77%]
2022-02-25T10:08:07.7277351Z        > ........................................................................ [ 79%]
2022-02-25T10:08:07.7281463Z        > ........................................................................ [ 81%]
2022-02-25T10:08:07.7284522Z        > ............................................................ssssssssssss [ 83%]
2022-02-25T10:08:07.7285048Z        > ssssssssssssssssssssssssssssssssssssssssssssssss........................ [ 85%]
2022-02-25T10:08:07.7288360Z        > .............................................................x.x.x.x.x.x [ 87%]
2022-02-25T10:08:07.7288772Z        > .x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x.x [ 89%]
2022-02-25T10:08:07.7291503Z        > ....s...sss......................ssssssssssss..................ssssssss. [ 92%]
2022-02-25T10:08:07.7299022Z        > .sssssssssss...ss...............................s.........s....s...../nix/store/r4xnq7cnjw20plycbqdl56988f5k3a0r-pytest-check-hook/nix-support/setup-hook: line 53:   185 Killed                  /nix/store/afi0ysqw20yiiw2gr2d28dx40bc4ddf8-python3-3.9.10/bin/python3.9 -m pytest -k "not test_class_omexml and not test_write_ome and not test_write_predictor"
```
anyone have any idea why the builder keeps getting killed? Does tifffile's test suite try to use lot of memory or something?

**Comment by samuela** [comment]
This appears to be fixed as of 1a268c42c8b0550f70da78c136171799481b0d97.
